lib/Jijo/Validation.hs

```diff
@@ -174,5 +178,19 @@ jValidateOptField fieldName vField o =
     mapJValidationReport (scopeJValidationReport (JPSField fieldName)) $
     vField field
 
+jValidateElements ::
```

Comment:
Shouldn't there also be `jValidateElement` for just one element?

Reply:
No, I don't think so. What would it do?

Reply:
```haskell
 jValidateField ::                     jValidateElement ::
   Text ->                               Int ->
   (j -> JValidation e a) ->             (j -> JValidation e a) ->
   HashMap.HashMap Text j ->             Vector j ->
   JValidation e a                       JValidation e a
```

Reply:
In fact, I also think there could be a `JArrayDefinition` that would be useful when working with JSON arrays containing different types. Aeson encodes tuples like that, but I assume there are APIs in the wild that also do that.

Reply:
Whether it should be implemented in this PR or left as an open issue is a separate question though.

Reply:
OK, sounds reasonable. I haven't thought about heterogeneous sequences. Yes, let's do it separately when the need arises.